src/Raylib/Core/Mouse.hpp

```diff
@@ -0,0 +1,111 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Mouse
+*/
+
+#ifndef MOUSE_HPP_
+#define MOUSE_HPP_
+
+#include "Vector2.hpp"
+
+extern "C"
+{
+#include "raylib.h"
+}
+
+namespace raylib
+{
+    namespace core
+    {
+        class Mouse {
+          public:
+            /// Constructor of the Mouse
+            Mouse();
+
+            /// Destructor of the Mouse
+            ~Mouse();
+
+            /// A wrapper function for the raylib function IsMouseButtonPressed.
+            ///
+            ///@param button The button to check.
+            ///
+            ///@return A boolean value.
+            bool isMouseButtonPressed(int button);
```

Comment:
Same remark as for the Cursor: all these methods could be static and omit the class name in the methods.